### PR TITLE
Linux: use correct column alignment for wide process commands

### PR DESCRIPTION
**Pull request by cgzones**

### CODE REVIEW

File: linux/LinuxProcess.c [hunk 1]
```diff
@@ -707,15 +707,18 @@ static void LinuxProcess_writeField(const Process* this, RichString* str, Proces
...
-         xSnprintf(buffer, n, "%-15.15s ", Process_isKernelThread(lp) ? kthreadID : "N/A");
+         procComm = Process_isKernelThread(lp) ? kthreadID : "N/A";
       }
-      break;
+      /* 15 being (TASK_COMM_LEN - 1) */
+      int c = RichString_appendnWide(str, attr, procComm, MINIMUM(15, strlen(procComm)));
+      RichString_appendChr(str, ' ', 16 - c);
+      return;
    }
    case PROC_EXE: {
```
Comment (BenBE) [inline linux/LinuxProcess.c:723]: `PROC_EXE` is affected from this issue just the same. as is the `CWD` column.

Similar may also apply for the DragonFlyBSD/FreeBSD `JAIL` column, and Solaris `ZONE` column.

**Review by BenBE — CHANGES_REQUESTED** [review]

**Review by BenBE — APPROVED** [review]

File: freebsd/FreeBSDProcessList.c [hunk 2]
```diff
@@ -411,7 +411,7 @@ static char* FreeBSDProcessList_readProcessName(kvm_t* kd, const struct kinfo_pr
 }
 
 static char* FreeBSDProcessList_readJailName(const struct kinfo_proc* kproc) {
-   char*  jname = NULL;
+   char*  jname;
```
Comment (BenBE) [inline freebsd/FreeBSDProcessList.c:414]: Keep initializion to `NULL` and only assign `"N/A""`if still at `NULL` upon trying to return.

**Review by BenBE — CHANGES_REQUESTED** [review]
Addressing the issue remarked upon by TravisCI.

**Review by BenBE — APPROVED** [review]

File: Process.c [hunk 3]
```diff
@@ -244,6 +244,11 @@ void Process_outputRate(RichString* str, char* buffer, size_t n, double rate, in
    }
 }
 
+void Process_printLeftAlignedField(RichString* str, int attr, const char* content, unsigned int width) {
+   int c = RichString_appendnWide(str, attr, content, MINIMUM(width, strlen(content)));
```
Comment (fasterit) [inline Process.c:248]: strlen will not account for wide chars (it essentially counts bytes)
So testcase:
```bash
echo -e '#!/bin/bash\nsleep 600' > 你好吗 && chmod +x 你好吗 && ./你好吗
```
and run htop with COMM as a visible column

Comment (BenBE) [reply Process.c:248]: Ref: https://github.com/htop-dev/htop/blob/master/RichString.c#L53

**Pull request merged**